### PR TITLE
Use \[...\] instead of $$...$$.

### PR DESCRIPTION
**Pull request by bangerth**
See https://tex.stackexchange.com/questions/503/why-is-preferable-to for the reason.

/rebuild

### CODE REVIEW

File: source/particle/integrator/interface.cc [hunk 1]
```diff
@@ -189,11 +189,11 @@ namespace aspect
                                "solves. If, for example, we call the numerical solution "
                                "of the ODE $\\tilde{\\mathbf x}_{k,h}(t)$, then the "
                                "error will typically satisfy a relationship like "
-                               "$$"
+                               "$["
```
Comment (tjhei) [inline source/particle/integrator/interface.cc:192]: Shouldn't this be \\[ ?

**Comment by bangerth** [comment]
Oh, doh. Fixed!

**Comment by tjhei** [comment]
I think you need to do ``\\[`` (which is what I had typed in my first comment, but github doesn't display ?!)

**Comment by bangerth** [comment]
Brain doesn't appear to work today. ... Fixed.

**Review by tjhei — APPROVED** [review]

**Pull request merged**